doc/source/parameters/cfd/post_processing.rst

```diff
@@ -252,5 +259,23 @@ This subsection controls the post-processing other than the forces and torque on
 
 * ``phase energy name``: name of the output file containing phase energies from Cahn-Hilliard simulations. The default file name is ``phase_energy``.
 
+* ``calculate volume phases``: outputs total volume of fluid phase and total volume of solid phase in CFD-DEM simulation. These volumes are computed as follow:
```

Comment:
should be phase volumes and not volume phases
like we calculate the volume of the phases and not the phase volume
:)